operator/src/main/resources/base-keycloak-deployment.yaml

```diff
@@ -29,14 +29,25 @@ spec:
         livenessProbe:
           exec:
             command:
-              - curl --head --fail --silent http://127.0.0.1:8080/health/live
+              - curl
+              - --head
+              - --fail
+              - --silent
+              - http://localhost:8080/health/live
           periodSeconds: 1
+          failureThreshold: 10
+          initialDelaySeconds: 30
         readinessProbe:
           exec:
             command:
-              - curl --head --fail --silent http://127.0.0.1:8080/health/ready
+              - curl
+              - --head
+              - --fail
+              - --silent
+              - http://localhost:8080/health/ready
           periodSeconds: 1
-          failureThreshold: 180
+          failureThreshold: 10
```

Comment:
This is effectively just 10 secs. What if the server is taking longer to boot?

Reply:
changed